packages/pds/tests/app-passwords.test.ts

```diff
@@ -90,7 +138,51 @@ describe('app_passwords', () => {
       },
     )
 
-    const attempt = appAgent.api.com.atproto.server.createAppPassword(
+    const priviAttempt = appAgent.api.com.atproto.server.getServiceAuth({
+      aud: 'did:example:test',
+    })
+    await expect(priviAttempt).rejects.toThrow('Bad token scope')
```

Comment:
I wonder if these lines were intended to land in a different test.

Reply:
ah nope - that's just an example route that allows only privileged app password auth

i added some comments to clarify 👌